From 534e7fac808a891af0ccdd72c47cc9b1ccfde460 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Fri, 20 Dec 2019 16:40:43 +0100
Subject: [PATCH] hdspmixer: Use __u32 and __u64 for RMS array types

Some variable types are referring to the field in ioctl struct, which
are actually __u32 or __u64 instead of uint32_t or uint64_t.
This inconsistency may result in the compile error.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 hdspmixer/src/HDSPMixerWindow.cxx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hdspmixer/src/HDSPMixerWindow.cxx b/hdspmixer/src/HDSPMixerWindow.cxx
index 342efb2..9efc25d 100644
--- a/hdspmixer/src/HDSPMixerWindow.cxx
+++ b/hdspmixer/src/HDSPMixerWindow.cxx
@@ -35,8 +35,8 @@ static void readregisters_cb(void *arg)
     hdsp_peak_rms_t hdsp_peak_rms;
     struct hdspm_peak_rms hdspm_peak_rms;
     bool isMADI = false;
-    uint32_t *input_peaks, *playback_peaks, *output_peaks;
-    uint64_t *input_rms, *playback_rms, *output_rms;
+    __u32 *input_peaks, *playback_peaks, *output_peaks;
+    __u64 *input_rms, *playback_rms, *output_rms;
     
     HDSPMixerWindow *w = (HDSPMixerWindow *)arg;
 
